recipes/devel/autotools.yaml

```diff
@@ -7,6 +7,7 @@ depends:
     - devel::autoconf-archive
     - devel::automake
     - devel::libtool
+    - devel::m4
```

Comment:
Why is this necessary? Previously m4 had to be available on the host or in the sandbox. Now it's available as tool. Why do we need another copy?